### DISCUSSION TITLE
Be Kept Up to Date on Future Path Changes

### DISCUSSION THREAD

**Discussion by emmahodcroft** [hodcroftlab/covariants#386] [General]
If you use any of the data files from CoVariants (for example, EUClusters_data.json, perCountryData.json, perCountryDataCaseCounts.json) and would like to be tagged in a reply update if the paths to these files change, please respond below listing which files you use.
We will tag your github username in replies of any future changes.

We aim to minimise this movement as much as possible, but we understand that alerting you to a change will be most convenient if files do move in the future.

If you prefer, you can also email covariants [a.t ]protonmail.com

**Comment by elysiumplain** [comment]
Awesome! Can you please add to the CC list. I can finally replace scripted polling with a direct file update thanks to this., especially with the slowdown of reporting urgency across particular districts.
